src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,175 @@ $pf-chart-color-black-200: $pf-color-black-400;
+$pf-chart-bullet--comparative-measure-line--error--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--comparative-measure-line--error--BorderWidth: 1;
+$pf-chart-bullet--zero-line--BorderColor: $pf-color-black-700;
+$pf-chart-bullet--zero-line--BorderWidth: 1;
+$pf-chart-bullet--negative-primary-measure--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--negative-primary-measure--BorderWidth: 2;
+
+// Candlestick
+$pf-chart-candelstick--data--stroke--Width: 1;
+$pf-chart-candelstick--data--stroke--Color: $pf-color-black-900; // grayscale theme
+$pf-chart-candelstick--candle--positive--Color: #fff; // grayscale theme
+$pf-chart-candelstick--candle--negative--Color: $pf-color-black-900; // grayscale theme
+
+// Chart
+$pf-chart-chart--BorderWidth: 1 solid;
+
+// Donut Chart
+$pf-chart-donut--threshold--first--Color: $pf-color-black-200;
+$pf-chart-donut--threshold--second--Color: $pf-color-black-300;
+$pf-chart-donut--threshold--third--Color: $pf-color-black-400;
+$pf-chart-donut--threshold--success--Color: $pf-chart-color-blue-300;
+$pf-chart-donut--threshold--warning--Color: $pf-chart-color-gold-400;
+$pf-chart-donut--threshold--danger--Color: $pf-color-red-100;
+$pf-chart-donut--label--subtitle--Color: $pf-color-black-500;
+
+// Error Bar
+$pf-chart-errorbar--BorderWidth: 8;
+$pf-chart-errorbar--data--BackgroundColor: transparent;
+$pf-chart-errorbar--data--Opacity: 1; // material theme
+$pf-chart-errorbar--data-stroke--Width: 2;
+$pf-chart-errorbar--data-stroke--Color: $pf-color-black-900; // grayscale theme
+
+// Group
+$pf-chart-group--grey-scale-color-scale: ($pf-color-black-900, $pf-chart-color-black-500, $pf-chart-color-black-400, $pf-chart-color-black-300, $pf-chart-color-black-200, $pf-chart-color-black-100); // grayscale theme
+
+// Legend
+$pf-chart-legend--gutter--Width: 20; // this is 10px in the grayscale theme
+$pf-chart-legend--orientation: "horizontal"; // this is vertical in material
+$pf-chart-legend--title--orientation: "top";
+$pf-chart-legend--data--type: "square"; // this is circle in the grayscale theme
+$pf-chart-legend--title--Padding: 2; // this is 5px in grayscale theme
+$pf-chart-legend--grey-scale--color-scale: ($pf-color-black-900, $pf-chart-color-black-500, $pf-chart-color-black-400, $pf-chart-color-black-300, $pf-chart-color-black-200, $pf-chart-color-black-100); // grayscale theme
+
+// Line Chart
+$pf-chart-line--data--BackgroundColor: transparent;
+$pf-chart-line--data--Opacity: 1; // material theme
+$pf-chart-line--data--stroke--Width: 2;
+$pf-chart-line--data--stroke--Color: $pf-color-black-900; // grayscale theme
+
+// Pie Chart
+$pf-chart-pie--Padding: 8; // 50 in grayscale theme
+$pf-chart-pie--data--Padding: 8; // this is 10px in grayscale theme
+$pf-chart-pie--data--stroke--Width: 1;
+$pf-chart-pie--data--stroke--Color: transparent; // grayscale theme
+$pf-chart-pie--labels--Padding: 8; // this is 20 in grayscale theme
+$pf-chart-pie--static--Height: 230; // 400 in grayscale theme
```

Comment:
@dlabrecq do you know what the difference is between static height, dynamic height, and just height, do you want vars for all three?

Reply:
Yes, we need all 3.

The height represents the overall SVG height. That can be used to change the scale of the SVG.

The static height is for the static portion of the chart. That is, the outer circle that represents the threshold -- that visualization doesn't change. 

The dynamic height is for the dynamic part of the chart. That is, the inner circle that changes dynamically based on the current value.

They are actually two separate pie charts that are used for the "donut utilization chart with thresholds".